dataikuapi/dss/ml.py

```diff
@@ -643,6 +660,335 @@ def __repr__(self):
+    """
+    def __init__(self, data):
+        self._internal_dict = data
+
+    @staticmethod
+    def create_from_expected_class(expected_valid_ratio, expected_class):
+        """
+        Creates an assertion condition from an expected valid ratio and  an expected class
+
+        :param float expected_valid_ratio: Ratio of valid rows needed for the assertion to pass
+        :param str expected_class: Class on which the `expected_valid_ratio` will be calculated
+
+        :rtype: :class:`dataikuapi.dss.ml.DSSMLAssertionCondition`
+        """
+        assertion_condition = DSSMLAssertionCondition({})
+        assertion_condition.expected_valid_ratio = expected_valid_ratio
+        assertion_condition.expected_class = expected_class
+        return assertion_condition
+
+    @staticmethod
+    def create_from_expected_range(expected_valid_ratio, expected_range):
+        """
+        Creates an assertion condition from an expected valid ratio and an expected range
+
+        :param float expected_valid_ratio: Ratio of valid rows to exceed for the assertion to pass
+        :param tuple(float,float) expected_range: Range of values (min, max) where the prediction will be considered as
+        valid for the `expected_valid_ratio`
+
+        :rtype: :class:`dataikuapi.dss.ml.DSSMLAssertionCondition`
+        """
+        assertion_condition = DSSMLAssertionCondition({})
+        assertion_condition.expected_valid_ratio = expected_valid_ratio
+        assertion_condition.expected_range = expected_range
+        return assertion_condition
+
+    def get_raw(self):
+        """
+        Gets the raw dictionary of the condition
+        :rtype: dict
+        """
+        return self._internal_dict
+
+    @property
+    def expected_class(self):
+        """
+        Returns the expected class on which the valid ratio will be calculated
+        :rtype: str
+        """
+        if "expectedClass" in self._internal_dict:
+            return self._internal_dict["expectedClass"]
+        else:
+            return None
+
+    @expected_class.setter
+    def expected_class(self, expected_class):
+        if self.expected_range is not None:
+            raise ValueError("Expected class and expected range can't be both set")
+        self._internal_dict["expectedClass"] = expected_class
+
+    @property
+    def expected_valid_ratio(self):
+        """
+        Returns the ratio of valid rows to exceed for the assertion to pass
+        :rtype: str
+        """
+        return self._internal_dict["successRatio"]
+
+    @expected_valid_ratio.setter
+    def expected_valid_ratio(self, expected_valid_ratio):
+        self._internal_dict["successRatio"] = expected_valid_ratio
+
+    @property
+    def expected_range(self):
+        """
+        Returns the expected range on which the valid ratio will be calculated
+        :rtype: tuple(float,float)
+        """
```

Comment:
To check: consistency with #95 where we expose a range as an object with min and max.